### ISSUE TITLE
Freeze when update found

### ISSUE THREAD

**Issue by 305jeff** [Epix37/Hearthstone-Deck-Tracker#499]
This has happened to me the last few updates...when I open the main window, it's grey and unclickable. The tracker continues working in HS just fine...but I cannot access the actual HSDT program until I force close it and restart. Upon restart, there's an update waiting. This has been consistent each time.

Thanks!

**Comment by Epix37** [comment]
That's if the tracker finds an update while running and not on start, correct?

**Comment by Cu8eR** [comment]
I get freezes when I start deck tracker... it takes about 1-2 seconds... everything is freezed - mouse cursor etc

**Comment by Epix37** [comment]
@Cu8eR do you have the "additional features" turned on? I would guess it's that combined with the huge increase in processed log data with this update.

**Comment by Cu8eR** [comment]
I dont think so... it freezes with stock default setting... but it is only for a 1-2 secs after start

**Comment by 305jeff** [comment]
Hi Epix37 - yes it happens with HSDT running. I've never had it freeze on start and the updates always pop up fine on start. I get the feeling that when I open HSDT from the system tray, there is an update window that I can't see, which disables my access to the actual program. Just a guess. Thanks!

**Comment by Cu8eR** [comment]
It freezes even when HS is not running

**Comment by 305jeff** [comment]
It sounds like these are separate issues. I've never had HSDT freeze except when an update is available and I open it while playing HS.

**Comment by Cu8eR** [comment]
No update available but it freezes - it is shorter but there is a freeze

**Comment by Epix37** [comment]
I think it's related to the logfile size at the start, rather than the version actually. These *are* different issues though. @305jeff is talking about a permanent freeze until restarted if I understood this corretly.

**Comment by 305jeff** [comment]
Correct - I have not found a way to "unfreeze" it once this happens. Eventually I just force close it, restart, and the update notification pops up.

**Comment by 305jeff** [comment]
Just happened again with the latest update. I was playing HS and noticed suddenly that after a game it didn't reload my deck...it just stayed with all the cards greyed out then started lighting them up as I drew them...kind of like reverse-mode. When I opened it from the notification pane, it was the same as before: everything greyed out, can't click or do anything.

Then I played around some more...

This time I found that if I shift+right click on it in the task bar and maximize it, suddenly the update notification is there. It remains even if I then restore it back to normal size. Hope this helps!
